### DISCUSSION TITLE
[General Question] About Window management

### DISCUSSION THREAD

**Discussion by didier31** [ArthurSonzogni/FTXUI#816] [General]
Hi,

excuse me by advance if my question is trivial but I'm wondering if FTXUI supports window overlapping.

Thanks a lot

**Comment by ArthurSonzogni** [comment]
Hello @didier31,

In some way, yes. See this example [(code)](https://github.com/ArthurSonzogni/FTXUI/blob/main/examples/component/window.cpp) [(demo)](https://arthursonzogni.github.io/FTXUI/examples/?file=component/window)

You can wrap any component inside a `Window`. They you can stack the window using the `Container::Stacked`.

**Reply by didier31** [reply]
Great ! Adopted.
Thanks again.
